packages/shared/lowPriorityWarning.js

```diff
@@ -5,47 +5,27 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import lowPriorityWarningWithoutStack from 'shared/lowPriorityWarningWithoutStack';
+import ReactSharedInternals from 'shared/ReactSharedInternals';
+
 /**
- * Forked from fbjs/warning:
- * https://github.com/facebook/fbjs/blob/e66ba20ad5be433eb54423f2b097d829324d9de6/packages/fbjs/src/__forks__/warning.js
- *
- * Only change is we use console.warn instead of console.error,
- * and do nothing when 'console' is not supported.
- * This really simplifies the code.
- * ---
  * Similar to invariant but only logs a warning if the condition is not met.
  * This can be used to log issues in development environments in critical
  * paths. Removing the logging code for production environments will keep the
  * same logic and follow the same code paths.
  */
 
-let lowPriorityWarning = function() {};
+let lowPriorityWarning = lowPriorityWarningWithoutStack;
 
 if (__DEV__) {
-  const printWarning = function(format, ...args) {
-    let argIndex = 0;
-    const message = 'Warning: ' + format.replace(/%s/g, () => args[argIndex++]);
-    if (typeof console !== 'undefined') {
-      console.warn(message);
-    }
-    try {
-      // --- Welcome to debugging React ---
-      // This error was thrown as a convenience so that you can use this stack
-      // to find the callsite that caused this warning to fire.
-      throw new Error(message);
-    } catch (x) {}
-  };
-
   lowPriorityWarning = function(condition, format, ...args) {
-    if (format === undefined) {
-      throw new Error(
-        '`lowPriorityWarning(condition, format, ...args)` requires a warning ' +
-          'message argument',
-      );
-    }
-    if (!condition) {
-      printWarning(format, ...args);
+    if (condition) {
+      return;
     }
+    const ReactDebugCurrentFrame = ReactSharedInternals.ReactDebugCurrentFrame;
+    const stack = ReactDebugCurrentFrame.getStackAddendum();
+    // eslint-disable-next-line react-internal/warning-and-invariant-args
+    lowPriorityWarningWithoutStack(false, format + '%s', ...args, stack);
```

Comment:
This fails the warning extraction script that fails here trying to strintify it. How did we solve that in the warning module?